src/basicfuns.jl

```diff
@@ -62,7 +62,7 @@ end
 """
 $(SIGNATURES)
 
-Return `x * exp(y)` for `y > -Inf`, or zero if `y == -Inf`.
+Return `x * exp(y)` for `y > -Inf`, or zero if `y == -Inf` or x == 0 and y is finite.
```

Comment:
To improve the display in the docs:
```suggestion
Return `x * exp(y)` for `y > -Inf`, or zero if `y == -Inf` or if `x == 0` and `y` is finite.
```